Readme no nersc account

stephjuneau opened the pull request:
I only modified the top-level README file. I added a Table of content, a section on external database access including what Stephen asked about TOPCAT access to the TAP handle. I also added a one-line section on DESI Data Acknowledgments pointing to the Data Doc website.

stephjuneau committed:
Update README.md

First test update to add TOC

stephjuneau committed:
Update README.md

Added TOC and content on database access without a NERSC account in the top-level README.

stephjuneau committed:
Update README.md

added stand-alone SPARCL instructions (separate from Data Lab)

stephjuneau committed:
Update README.md

small changes of wording

stephjuneau committed:
Update README.md

fixes internal link to external databases

weaverba137 commented on README.md:
```diff
@@ -5,6 +5,13 @@ Welcome to the repository designed for tutorials that guide you through the proc
 * Tutorials that were demonstrated at specific DESI meetings, each segregated under directories marked by the date of the meeting. For example, meetings/Dec2020.
 * Tutorials designed around wide-ranging subjects (like onskydata) or specific packages (such as fiberassign, redrock).
 
+### Table of Content
```
"Contents"

stephjuneau replied:
Fixed. Thanks!

weaverba137 commented on README.md:
```diff
@@ -116,6 +123,41 @@ Tutorials hosted in other packages include (but haven't been recently vetted):
+
+# Acknowledgments
+
+Any use of DESI data whether via NERSC or external databases requires [DESI Data Acknowledgments](https://data.desi.lbl.gov/doc/acknowledgements/).
```
Double-check the spelling: https://github.com/desihub/desidatadocs/blob/edr/docs/acknowledgments.md

sbailey replied:
for context, there is a British vs. American spelling difference for "acknowledgments", and even the pubboard acknowledgments wiki page used both on a single page.  Current main uses British acknowledgements (with an extra "e") but the edr branch soon to be merged will switch to the American spelling "acknowledgments" as the final URL to use.  We need to check this on the NOIRLab websites too!

stephjuneau replied:
Yes, we also had (have?) both versions of the spelling on the Data Lab website. After figuring out which is the American spelling, we have so far fixed it on the SPARCL website (https://astrosparcl.datalab.noirlab.edu/sparc/acknowledgments/).

That said, we won't be able to change the SPARCL website for a while (complicated -- it's tied to code releases) so are you able to add a re-direct such that "https://data.desi.lbl.gov/doc/acknowledgements/" re-directs to "https://data.desi.lbl.gov/doc/acknowledgments/" meaning that either spelling would not cause an error? (if not, a hack would be to create two identical "acknowledgments.md" and "acknowledgements.md" pages for now, which is of questionable taste but might work in a pinch).

stephjuneau replied:
We have used the Acknowledgments in many places so tracking it all down might take a while even on the Data Lab website (which we can still modify today, unlike the SPARCL website).

stephjuneau replied:
@sbailey  could you confirm that I should change the URL in the README (and everywhere we still possibly can) to "https://data.desi.lbl.gov/doc/acknowledgments/" without the British "e"?

sbailey replied:
yes, please change it everywhere you can to the American spelling https://data.desi.lbl.gov/doc/acknowledgments/ (which does not yet exist).  I'll look into our options for also supporting redirect for the British spelling.

sbailey replied:
https://data.desi.lbl.gov/doc/acknowledgments/ now does exist as the preferred link to use, and desihub/desidatadocs#60 will restore support for the older spelling for pre-deployed sites using the older URL.

stephjuneau replied:
OK great. Marking as resolved.

weaverba137 reviewed (changes requested):
I found a couple of spelling errors.

forero reviewed (approved):
Aside from the spelling issue, I believe these extra bits in the README are useful.

stephjuneau committed:
Update README.md

fixed "Contents" and "acknowledgments" in the URL

forero commented:
@stephjuneau Please go ahead and merge this branch whenever you are ready. I will proceed merging the database branch afterwards.

stephjuneau committed:
Create README.md

New README in getting_started/ which is linking to the other .md files at the moment in case they are also linked from elsewhere.

stephjuneau commented:
OK so I made the two small changes requested by Ben (removed the extra "e" in acknowledgments URL). I have also just added a new README to the getting_started/ folder. This links to the other two previous .md files but we could incorporate them directly if preferred? @akremin will need to update the link to the EDR paper repo. (@forero what do you think of the new README file? do the links to the other md files work for you?)

stephjuneau commented:
I was trying to let Ben know that I made his two requested changes but I accidentally re-requested a review. Not sure that was needed but I'll wait to hear a minimum of one approval before merging (whoever sees this first :-) )

stephjuneau committed:
Update README.md

fixed links to .md files

weaverba137 commented on README.md:
````diff
@@ -116,6 +123,41 @@ Tutorials hosted in other packages include (but haven't been recently vetted):
+```
+The client can be loaded within a Python session or program via:
+```
+>> from sparcl.client import SparclClient 
````
The standard Python prompt has three arrows: `>>>`.

stephjuneau replied:
fixed.

weaverba137 commented on getting_started/README.md:
```diff
@@ -0,0 +1,22 @@
+
+### DESI Helpdesk
+
+If you have a question, please visit the [DESI User Forum](https://help.desi.lbl.gov)
```
Period at the end of the sentence.

stephjuneau replied:
fixed.

stephjuneau commented:
Hi @weaverba137 are you OK with me merging this PR? I've made both fixes (and added a README in case you want to see it: https://github.com/desihub/tutorials/blob/readme-no-nersc-account/getting_started/README.md) **Edit:** Jaime requested to merge this one first before he merges the database notebook branch)

weaverba137 commented:
I found some additional problems, but they can be fixed quickly.

stephjuneau committed:
Update README.md

fixed missing period

stephjuneau committed:
Update README.md

fixed python prompt

stephjuneau commented:
OK, I made both new fixes and will merge. Thanks for the reviews!
